modules/templates/proc-adding-templates.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-adding-templates_{context}"]
+= Adding an existing template to {product} by using the Template Editor
+
+You can add an existing template to your {product} instance by using the Template Editor.
+
+.Prerequisites
+
+* You have created a directory that contains template files.
+* You have created at least one template YAML file in the directory.
+
+.Procedure
+
+. Access the Template Editor by using one of the following options:
++
+image::rhdh/template-editor.png[Template Editor]
+** Open the URL `\https://<rhdh_url>/create/edit` for your {product} instance.
+** Click *Create...* in the navigation menu of the {product} console, then click the overflow menu button and select *Template editor*.
+. Click *Load Template Directory*. This allows you to upload a template from a local directory, by navigating to the directory and clicking *Select*.
```

Comment:
Can you add please a warning to mention that the button `Load Template Director` is available using google Chrome but not with Firefox. Backstage ui help mentions "Only supported in some Chromium-based browsers"

Reply:
Good call, I will add something for this